mobile/examples/image_classifications/android/README.md

```diff
@@ -1,46 +1,42 @@
 # ONNX Runtime Mobile image classification Android sample application
 
 ## Overview
-This is an example application for [ONNX Runtime](https://github.com/microsoft/onnxruntime) on Android. The demo app uses image classification which is able to continuously classify the objects it sees from the device's camera in real-time and displays the most probable inference result on the screen.
+This is an example application for [ONNX Runtime](https://github.com/microsoft/onnxruntime) on Android. The demo app uses image classification which is able to continuously classifying the objects it sees from the device's camera in real-time and displays the most probable inference results on the screen.
```

Comment:
>which is able to continuously classifying the objects it sees from the device's camera in real-time and displays the most probable inference  [](http://example.com/codeflow?start=143&length=141)

'continuously classify' was correct. 

or

'The demo app uses image classification that continuously classifies...'